src/components/modalWrapper/ModalWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactNode, useEffect, useRef } from "react";
+import { createPortal } from "react-dom";
+import s from "./modalWrapper.module.scss";
+
+interface ModalWrapperProps {
+  children: ReactNode;
+}
+
+function ModalWrapper({ children }: ModalWrapperProps) {
+  const modalRootRef = useRef(document.getElementById("modal"));
```

Comment:
No getElementById, we forget about such things in React

Reply:
fix